db/main/migrate/20170203124000_add_constraint_for_owner_id_to_subscriptions.rb

```diff
@@ -0,0 +1,6 @@
+class AddConstraintForOwnerIdToSubscriptions < ActiveRecord::Migration
+  def up
+    execute "ALTER TABLE subscriptions ADD CONSTRAINT subscriptions_owner_id UNIQUE (owner_id, owner_type);"
```

Comment:
The constraint should be unique owner_id and owner_typer where status == 'subscribed' to match the rails constraint.